docs/features.md

```diff
@@ -2,4 +2,35 @@ Click the below for a walk through of the product functionality:
 
 [![PostHog demo](_media/posthog-demo.png)](https://youtu.be/vGy9gzMAAes-Y?t=35s "PostHog Demo Video")
 
-There is a detailed table of the functionality on the [pricing](/pricing) page.
\ No newline at end of file
+There is a detailed table of the functionality on the [pricing](/pricing) page.
+
+Below is a table of PostHog features and how this compares with the functionality in common alternatives.
+
+
+|  Feature 	|   PostHog	| Amplitude  	|  Mixpanel 	| Heap  	|
```

Comment:
We should put PostHog (Community Edition) as our name here to make it clear why we don't cover everything (ie the ab testing)

Reply:
I was thinking of adding 2 separate column's "Growth" and "Ultimate" however if this aim of this is to just show where parity is then anything in "Ultimate" would be part of PostHog, i.e. when we have built that it would get a tick.

- I worry it becomes less clear if that was the case and this is just doubling the table here - https://handbook.posthog.com/#/pricing